src/musicbrainz-api.ts

```diff
@@ -363,6 +363,15 @@ export class MusicBrainzApi {
     return this.lookupEntity<mb.IUrl, UrlIncludes>('url', urlId, inc);
   }
 
+  /**
+   * Lookup Event
+   * @param eventId Event MBID
+   * @param inc EventIncludes
```

Comment:
I suggest clear parameter names
```suggestion
   * @param eventIncludes EventIncludes
```